src/main/webapp/css/srp-style.css

```diff
@@ -0,0 +1,41 @@
+.srp-container {
```

Comment:
Can you rename srp-style.css to srp_style.css? Filenames should be lowercase_underscore.